samples/word/06-lists/insert-list.yaml

```diff
@@ -4,52 +4,63 @@ description: Inserts a new list into the document.
 author: OfficeDev
 host: WORD
 api_set:
-    WordApi: 1.1
+    WordApi: 1.3
 script:
     content: |-
-        $("#insert-controls").click(insertList);
-        $("#setup").click(setup);
+        $("#insert-controls").click(() => tryCatch(insertList));
+        $("#setup").click(() => tryCatch(setup));
 
-        function insertList() {
+        async function insertList() {
             //this example starts a new list stating with the second paragraph.
-            Word.run(function (context) {
-                var paragraphs = context.document.body.paragraphs;
+            await Word.run(async (context) => {
+                let paragraphs = context.document.body.paragraphs;
                 paragraphs.load();
```

Comment:
Let's make this a non-empty load.

Reply:
we need no scalars, just local fixed it.